docs/products/mysql/howto/migrate-database-mysqldump.rst

```diff
@@ -0,0 +1,107 @@
+Migrate your MySQL data using ``mysqldump``
```

Comment:
I would say the perfect title is "Backup and restore MySQL data using ``mysqldump``"